packages/form/addon/components/cf-field/label.hbs

```diff
@@ -3,11 +3,18 @@
     {{@field.question.raw.label}}
   </span>
 
-  {{#if (and (not @field.question.isCalculated) @field.optional)}}
-    <span
-      class="uk-margin-small-left uk-text-muted uk-text-lowercase uk-text-normal"
-    >({{t "caluma.form.optional"}})</span>
-  {{/if}}
+  {{#unless @field.question.isCalculated}}
+    {{#if (and @field.optional (not this.useMandatoryAsterisk))}}
+      <span
+        class="uk-margin-small-left uk-text-muted uk-text-lowercase uk-text-normal"
+      >({{t "caluma.form.optional"}})</span>
+    {{else if (and (not @field.optional) this.useMandatoryAsterisk)}}
+      <span
+        title={{t "caluma.form.mandatory"}}
+        class="uk-text-muted uk-text-lowercase uk-text-normal mandatory-asterisk"
```

Comment:
I think the class `mandatory-asterisk` is obsolete, right?

Reply:
Without any other context, this class is useless, right. I added it, so that a user of caluma can provide specific css styles for that asterisk without wirting crazy css selectors.

It was just an idea to add it like that, do you have any suggestions on it or should I remove it?

Reply:
That is a really valid point I didn't think about. In that case we should keep it but rename the class to be consistent with other custom classes we use.

In general we *try* to use [BEM](https://getbem.com/) as our naming convention. I'd propose to use `.cf-label__manatory-asterisk` as class name.